PermanentOrg/web-app
# PER-9142: Add Default Access Role to Share Links (feat. PER-9180: Add Storybook)

This PR adds the ability to set the default access role of a share link in the sharing dialog. Also it installs Storybook in this repo and includes a story for the SharingDialogComponent and the AnnouncementComponent (this last one was just an initial test, but it's a usable story so why would I delete it).

A large chunk of the new files in this PR (those in `src/stories/` and `.storybook/`) are just generated from Storybook's installer CLI and can be ignored. Right now I"d like to keep the example stories in here just to serve as living documentation as we get used to using Storybook with this project.

Steps to test:
1. Open the sharing dialog on an item in the Archive Manager
2. Create a share link if it doesn't exist already
3. Verify that the Default Access Role property is there. Try setting it to something besides the default, and verify that it saves and works properly. 
1. Run `npm run storybook` to test out Storybook!

Resolves PER-9142, and features *special guest ticket* PER-9180 as well.
